src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilder.cs

```diff
@@ -345,9 +345,14 @@ internal static unsafe void SetConstantValue(ModuleBuilder module, int tk, Type
 
                 switch (corType)
                 {
+                    case CorElementType.ELEMENT_TYPE_BOOLEAN:
```

Comment:
Change in this file should be undone as well. It is not part of parsing/formatting/conversions. It is emitting types that is a counterpart of type loading. Both should be on the same plan.

Reply:
(Also, please bring back the matching test.)